Service/InstallationService.php

```diff
@@ -1302,7 +1307,47 @@ private function createCronjobs(array $actions = []): array
 
         return $cronjobs;
     }//end createCronjobs()
-
+    
+    /**
+     * This function creates applications with the given $applications data.
+     * Each application in this array should have an organization = reference.
+     *
+     * @param array $applicationsData An array of arrays describing the application objects we want to create or update.
+     *
+     * @return array An array of applications.
+     */
+    private function createApplications(array $applicationsData): array
+    {
+        $orgRepository = $this->entityManager->getRepository('App:Organization');
+        
+        foreach ($applicationsData as $key => &$applicationData) {
+            if (isset($applicationData['$id']) === false) {
+                $this->logger->error("Can't create an Application without '\$id': 'reference'", ['applicationData' => $applicationData]);
+                unset($applicationsData[$key]);
+                
+                continue;
+            }
```

Comment:
//end if

Reply:
end if is only needed if the if statement is long